testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/WildcardReadsTestCase.java

```diff
@@ -350,7 +350,7 @@ public void testWildcardHostRootQuery() {
         assertNotNull(resp.toString(), slaveResult);
 
         // Now limit the result to secondary hosts
-        op.get(WHERE, MASTER).set(false);
```

Comment:
Hi @parsharma , this change is not expected on this commit since it is not related to the property renaming, the cause of this change is the renaming of the default hostnames.

In any case, this is the source of the errors you are still getting on the integration Jobs. It's a new case you need to handle differently on the test case.

This operation is querying the mixed domain host resources by filtering the results using "primary = false" where condition. As part of problematic language usage work, we have already renamed the attribute "master" on the "host" resource to "primary", see https://issues.redhat.com/browse/WFCORE-5844. That means this operation in this scenario no longer works as the test case expects because the legacy host, which is part of the mixed domain, uses the attribute "master". The host attribute "primary" doesn't exist on the legacy host.

You have to adapt this test case to handle property this situation. The simplest solution is to execute two operations, one using "primary = false" where condition, and the other using "master = false" where condition.
You should assert that "primary = false" returns empty, and "master = false" returns 1.

A more elegant solution would be to create a composite operation and assert the same.

This where condition is used in other parts of the test, so maybe you can get a common code extracted so it can be invoked from the other tests done on this class.